tests/data/results/www.jamieoliver.com/recipes-eggs-recipes-simple-cheese-omelette.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+return [
+    'author' => 'Jamie Oliver',
+    'categories' => [
+        'Breakfast',
+        'Jamie\'s Food Revolution',
+        'Eggs',
+        'Brilliant breakfast recipes',
+        'Family one-pan recipes',
+        'Mains',
+    ],
+    'cookingMethod' => '',
+    'cookTime' => '',
+    'cuisines' => [
+        'Gluten-free',
+        'Vegetarian',
+    ],
+    'description' => 'This is a great quick meal at any time of the day. Eggs are a fantastic, affordable source of protein, helping our muscles to grow and repair and helping us to feel fuller for longer. Serve your omelette with wholemeal bread and a simple tomato salad for a healthy, balanced meal.',
+    'image' => 'http://cdn.jamieoliver.com/recipe-database/xtra_med/0siQaM1MqTv8KwCQYOv41F.jpg',
+    'ingredients' => [
+        '2 large free-range eggs',
+        'olive oil',
+        '10 g Cheddar cheese',
+    ],
+    'instructions' => [
+        'Crack the eggs into a mixing bowl, season with a pinch of sea salt and black pepper, then beat well with a fork until fully combined.',
+        'Place a small non-stick frying pan on a low heat to warm up.',
+        'Grate the cheese onto a board and set aside.',
+        'Add ½ tablespoon of oil to the hot pan, then carefully pour in the eggs.',
+        'Tilt the pan to spread them out evenly, using a fork to swirl the eggs around the pan a little.',
+        'When the omelette begins to cook and firm up, but still has a little raw egg on top, sprinkle over the cheese.',
+        'Using a spatula, ease around the edges of the omelette, then fold it over in half.',
+        'When it starts to turn golden brown underneath, remove the pan from the heat and slide the omelette onto a plate. Delicious with a tomato salad and wholemeal bread.',
+        'PRINT THIS RECIPE (UK) PRINT THIS RECIPE (USA)',
```

Comment:
Maybe this should be caught and removed, it'd have to be via strpos or similar, they've chucked it into the method steps.

Reply:
Agreed - from a quick google search it looks like there are only a handful of recipes with a print link in the instructions - I would probably filter using stripos to avoid casing issues.

Reply:
Now filtering steps containing 'print this recipe'